problem1.java

```diff
@@ -0,0 +1,50 @@
+import java.util.LinkedList;
+import java.util.Queue;
+
+class rottingOranges{
```

Comment:
Please describe your approach briefly.

Reply:
I didn't get your question here. What do you want me to describe here??